### PR TITLE
feat(api): add tags issues

### PR DESCRIPTION
**Pull request by djabarovgeorge**
### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: apps/api/src/app/workflows-v2/usecases/process-workflow-issues/process-workflow-issues.usecase.ts [hunk 1]
```diff
@@ -123,28 +123,66 @@ export class ValidateAndPersistWorkflowIssuesUsecase {
...
+
+    if (!tags.length) {
+      return;
+    }
+
+    const tagsIssues: RuntimeIssue[] = [];
+
+    const hasDuplications = tags.length !== new Set(tags).size;
+    if (hasDuplications) {
+      tagsIssues.push({ issueType: WorkflowIssueTypeEnum.DUPLICATED_VALUE, message: 'Duplicated tags' });
```
Comment (djabarovgeorge) [inline apps/api/src/app/workflows-v2/usecases/process-workflow-issues/process-workflow-issues.usecase.ts]: i wonder if we should add the duplication values here or is the generic message will be enough.

File: apps/dashboard/src/components/workflow-row.tsx [hunk 2]
```diff
@@ -95,7 +95,7 @@ export const WorkflowRow = ({ workflow }: WorkflowRowProps) => {
             </TableCell>
           </TooltipTrigger>
           <TooltipPortal>
-            <TooltipContent>{new Date(workflow.updatedAt).toUTCString()}</TooltipContent>
```
Comment (djabarovgeorge) [inline apps/dashboard/src/components/workflow-row.tsx:98]: apologies webstorm did not push this change commit to [my last PR](https://github.com/novuhq/novu/pull/6948) 😅

File: apps/api/src/app/workflows-v2/usecases/process-workflow-issues/process-workflow-issues.usecase.ts [hunk 3]
```diff
@@ -15,18 +15,18 @@ import {
...
 } from '@novu/dal';
 import { Injectable } from '@nestjs/common';
-import { ValidateWorkflowCommand } from './validate-workflow.command';
-import { WorkflowNotFoundException } from '../../exceptions/workflow-not-found-exception';
+import { ProcessWorkflowIssuesCommand } from './process-workflow-issues.command';
 import { ValidateControlValuesAndConstructPassableStructureUsecase } from '../validate-control-values/build-default-control-values-usecase.service';
+import { WorkflowNotFoundException } from '../../exceptions/workflow-not-found-exception';
 
 @Injectable()
-export class ValidateAndPersistWorkflowIssuesUsecase {
```
Comment (djabarovgeorge) [inline apps/api/src/app/workflows-v2/usecases/process-workflow-issues/process-workflow-issues.usecase.ts:23]: rename to a shorter name and move to dedicated usecase folder

File: apps/api/src/app/workflows-v2/usecases/process-workflow-issues/process-workflow-issues.command.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import { EnvironmentWithUserObjectCommand, GetPreferencesResponseDto } from '@novu/application-generic';
 import { ControlValuesEntity, NotificationTemplateEntity } from '@novu/dal';
 
-export class ValidateWorkflowCommand extends EnvironmentWithUserObjectCommand {
```
Comment (djabarovgeorge) [inline apps/api/src/app/workflows-v2/usecases/process-workflow-issues/process-workflow-issues.command.ts:4]: align the name with the usecase

File: apps/api/src/app/workflows-v2/usecases/process-workflow-issues/process-workflow-issues.usecase.ts (hunk 1 above)
Comment (djabarovgeorge) [reply apps/api/src/app/workflows-v2/usecases/process-workflow-issues/process-workflow-issues.usecase.ts]: never mind i added those

**Review by SokratisVidros — APPROVED** [review]

**Review by SokratisVidros — APPROVED** [review]

**Pull request merged**